crates/re_viewer/src/ui/view_spatial/scene/scene_part/mod.rs

```diff
@@ -42,7 +42,8 @@ pub trait ScenePart {
 
 /// Computes the instance hash that should be used for picking (in turn for selecting/hover)
 ///
-/// Takes into account the currently the object properties, currently highlighted objects, and number of instances.
+/// TODO(andreas): Resolve the picking upon picking instead of doing it ahead of time here. This will speed up things.
```

Comment:
"Resolve the picking upon picking" :question:

Reply:
the ai is taking over!